authorize/grpc.go

```diff
@@ -134,38 +136,12 @@ func (a *Authorize) getEvaluatorRequestFromCheckRequest(
 	}
 	p := a.getMatchingPolicy(requestURL)
 	if p != nil {
-		for _, sp := range p.SubPolicies {
-			req.CustomPolicies = append(req.CustomPolicies, sp.Rego...)
-		}
-	}
-
-	ca, err := a.getDownstreamClientCA(p)
-	if err != nil {
-		return nil, err
+		req.Policy = p
```

Comment:
if p is not nil, what would policy be? Could we instead do `req.Policy = a.getMatchingPolicy(requestURL)` above?